src/combinatorics/colex.lean

```diff
@@ -318,30 +287,75 @@ begin
     rw z hx },
 end
 
-/-- For subsets of ℕ, we can show that colex is equivalent to binary. -/
-lemma sum_sq_lt_iff_lt (A B : finset ℕ) : A.sum (pow 2) < B.sum (pow 2) ↔
+/-- Colex doesn't care if you remove the other set -/
+@[simp] lemma sdiff_le_sdiff_iff_le [linear_order α] (A B : finset α) :
+  (A \ B).to_colex ≤ (B \ A).to_colex ↔ A.to_colex ≤ B.to_colex :=
+by rw [le_iff_le_iff_lt_iff_lt, sdiff_lt_sdiff_iff_lt]
+
+lemma empty_to_colex_lt [linear_order α] {A : finset α} (hA : A.nonempty) :
+  (∅ : finset α).to_colex < A.to_colex :=
```

Comment:
Should `(∅ : finset α).to_colex` be a `has_bot` instance? Can you provide an `order_bot` instance with this result?

Reply:
Done.